src/test/java/com/google/idea/perf/CallTreeTest.kt

```diff
@@ -25,11 +25,17 @@ class CallTreeTest {
         override val tracepoint: Tracepoint,
         override val callCount: Long,
         override val wallTime: Long,
+        override val maxCallTime: Long,
         childrenList: List<Tree> = emptyList()
     ) : CallTree {
         override val children = childrenList.associateBy { it.tracepoint }
     }
 
+    init {
+        // Enforce LF line endings for developers on Windows.
+        System.setProperty("line.separator", "\n");
```

Comment:
I guess this will affect other tests too, since the property persists for the life of the JVM.
Maybe it's harmless, although I also wonder if there's another way.
Not worth worrying about now though!

Reply:
Sure! I'll keep an eye on it whenever I start making new unit tests.